### DISCUSSION TITLE
Игра Пакман

### DISCUSSION THREAD

**Discussion by dimmkin** [dimmkin/tpro-ddz-packman#9] [General]
Сейчас буду создавать макет визуальной части процесса игры, будет интересно ваше мнение насчет этого
И отправлю вам скриншоты начального игрового меню и меню после нажатия на кнопку START

**Comment by dimmkin** [comment]
![image](https://github.com/dimmkin/tpro-ddz-packman/assets/120401947/50bc3b7a-8f93-42e5-9a3f-b3649c3cad31)


![image](https://github.com/dimmkin/tpro-ddz-packman/assets/120401947/3141c600-d047-480d-a8cc-0db8f537fdb1)

**Comment by dimmkin** [comment]
В таком же стиле хочу нарисовать визуальную часть игрового процесса. Как вам?)

**Reply by dno442** [reply]
Найс!

**Comment by SstraD-s** [comment]
Парни, первые две карты, на оценку


![Снимок экрана от 2023-11-13 22-48-15](https://github.com/dimmkin/tpro-ddz-packman/assets/77076210/15420943-5463-47ce-8ff2-258a9b0870e0)




![Снимок экрана от 2023-11-13 23-40-32](https://github.com/dimmkin/tpro-ddz-packman/assets/77076210/1dea5e6a-ab05-40b0-a050-0dfd4ed482df)

**Reply by dimmkin** [reply]
Прикольно)

**Comment by SstraD-s** [comment]
А вот и третья карта подъехала)

![Снимок экрана от 2023-11-14 00-23-05](https://github.com/dimmkin/tpro-ddz-packman/assets/77076210/0b5aec48-9887-4fc6-8c7b-f85918f32ca4)

**Comment by dimmkin** [comment]
Только тебе не кажется, что карта не должна быть обрамлена полностью, то есть, игрок может выходить за края карты и появляться на противоположной стороне

**Reply by dimmkin** [reply]
В условии написано так

**Reply by SstraD-s** [reply]
а, не заметил этого. Ну ладно, там исправить  - 1 на 0 поменять  <:{)
